main.py

```diff
@@ -35,7 +38,7 @@ async def auto_send():
 
 if __name__ == "__main__":
     try:
-        bot.run(BOT_TOKEN)
+        bot.run(BOT_TOKEN, log_handler=None)
```

Comment:
Could you describe why **you need** to pass `log_handler=None`? (I, myself, know but you know it'll be useful for other reviewers!)
some documentation links should really help!

Reply:
We need to pass None as the log_handler in order to propagate logs to the root (to process them).

Reply:
to other reviewers: you can resolve this whenever you want!